master/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -73,20 +67,3 @@ independently.
 By default, the manifest expects an etcd proxy to be running on each
 Kubernetes node at `http://127.0.0.1:2379`.
 
-
-### Requirements / Limitations
-
-- This install assumes no other pod network configurations have been installed
-  in /etc/cni/net.d (or equivalent directory).
-- The Kubernetes cluster must be configured to provide service account tokens to pods.
-- kubelets must be started with `--network-plugin=cni` and have
-  `--cni-conf-dir` and `--cni-bin-dir` properly set.
-  - If using kubeadm these will be set by default.
-- The Kubernetes controller manager must be started with
```

Comment:
Just noting - this bit was a requirement _for this doc only_, so we shouldn't try to generalize this and move it into another place.